src/locales/en/messages.json

```diff
@@ -4556,5 +4670,23 @@
   },
   "migratedKeyConnector": {
     "message": "Migrated to Key Connector"
+  },
+  "paymentSponsored": {
+    "message": "Please provide a payment method to associate with the organization. Don't worry, we won't charge you anything unless you select additional features or your sponsorship expires. "
+  },
+  "removedSponsorshipSuccess": {
+    "message": "Success! Sponsorship removed."
```

Comment:
Should we include `Success!` here? The toast is going to be green and I don't believe we use this format for other notifications.

Reply:
I agree and I already have a message for it without `Success!`